provider/src/main/scala/com/actian/spark_vector/provider/RequestHandler.scala

```diff
@@ -16,23 +16,23 @@
 package com.actian.spark_vector.provider
 
 import java.nio.channels.SocketChannel
-import java.util.concurrent.atomic.AtomicLong
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.util.{ Failure, Success }
 
 import org.apache.spark.Logging
-import org.apache.spark.sql.{ DataFrame, SQLContext, SaveMode }
+import org.apache.spark.sql.{ DataFrame, Row, SQLContext, SaveMode }
+import org.apache.spark.sql.types.StructType
 
 import com.actian.spark_vector.datastream.reader.DataStreamReader
 import com.actian.spark_vector.datastream.writer.DataStreamWriter
-import com.actian.spark_vector.loader.command.sparkQuote
-import com.actian.spark_vector.sql.VectorRelation
-import com.actian.spark_vector.util.ResourceUtil.closeResourceAfterUse
+import com.actian.spark_vector.sql._
```

Comment:
if it's possible please (and not to long) import explicitly the classes you need